daemon/snap.go

```diff
@@ -54,31 +55,53 @@ func snapDate(info *snap.Info) time.Time {
 	return st.ModTime()
 }
 
+func publisherName(st *state.State, info *snap.Info) (string, error) {
+	publisher := ""
+	if info.SnapID != "" {
```

Comment:
(nitpick): it looks like this could be written as:
```
if info.SnapID == "" { return "", nil }
...
```
I.e. avoiding one level of nesting.